TensorInfo[] tensors
uint32 batchSize
uint8[] data
